### PR TITLE
Added option for select all fields in task variable and removed show in list option

### PR DESCRIPTION
**Pull request by shuhaib-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3101
Issue Type: BUG/ FEATURE

## Pull Request Changes

In this pull request, the following modifications have been made:

- **Feature Addition:** Implemented the ability to select all form fields for task variables.
- **UI Update:** Removed the "show in list" option from the user interface.
- **Database Adjustment:** Added a `select_all_field` column to the mapper table. This column retains the selection status for all form fields, persisting in the database for future use.
- **Migration File:** Created a migration file to implement these changes.

Moreover, I ran `isort` and `black` on the `src/formslfow_api` directory, resulting in formatting changes in some Python files.

 # Screenshots (if applicable)
image
image

### CODE REVIEW

File: forms-flow-api/src/formsflow_api/services/application.py [hunk 1]
```diff
@@ -497,9 +498,11 @@ def fetch_task_variable_values(task_variable, form_data):
         if task_variable and form_data:
             task_keys = [val["key"] for val in task_variable]
             variables = {
-                key: {"value": json.dumps(form_data[key])}
-                if isinstance(form_data[key], (dict, list))
-                else {"value": form_data[key]}
+                key: (
```
Comment (sumesh-aot) [inline forms-flow-api/src/formsflow_api/services/application.py:501]: Just to confirm, the changes to push all variables to BPM is not part of this PR right ?

File: forms-flow-api/src/formsflow_api/models/form_process_mapper.py [hunk 2]
```diff
@@ -52,6 +52,7 @@ class FormProcessMapper(AuditDateTimeMixin, AuditUserMixin, BaseModel, db.Model)
     task_variable = db.Column(JSON, nullable=True)
     version = db.Column(db.Integer, nullable=False, default=1)
     description = db.Column(db.String, nullable=True)
+    selected_all_field = db.Column(db.Boolean, nullable=True, default=False)
```
Comment (auslin-aot) [inline forms-flow-api/src/formsflow_api/models/form_process_mapper.py:55]: Could you please include the purpose of the field in the comment parameter?

**Review by auslin-aot — APPROVED** [review]

File: forms-flow-api/src/formsflow_api/services/application.py (hunk 1 above)
Comment (auslin-aot) [reply forms-flow-api/src/formsflow_api/services/application.py:501]: When the "select all fields" option is enabled, all form fields are captured in the task_variable within the form_process_mapper. During form submission, the fields will be pushed to BPM based on the task_variable found in the mapper. So here all fields should be pushed to bpm

**Review by sumesh-aot — APPROVED** [review]
